_pages/_resources/disability-rights-guide.md

```diff
@@ -57,7 +57,7 @@ Complaints of title II violations may be filed with the Department of Justice wi
 >Washington, D.C. 20530
 >{{site.contact.tollfree}} (voice)
 >{{site.contact.tty}} (TTY)
->[ADA.gov](https://www.ada.gov)
+>[ADA.gov]({{'/index' | relative_url}})
```

Comment:
I think we want these to be `{{ '/' | relative_url }}` rather than using /index

Reply:
thank you, @kewitham.  changes made.